spec/support/shared/examples/msf/db_manager/cred.rb

```diff
@@ -1,4 +1,9 @@
 RSpec.shared_examples_for 'Msf::DBManager::Cred' do
+
+  if ENV['REMOTE_DB']
+    before {skip("Awaiting cred port")}
+  end
```

Comment:
@mkienow-r7 This might be a bit of a grave dig; But I don't suppose you know the latest information on this `skip` message? 😄 

Were the creds API ever ported to the REMOTE_DB?

Reply:
@adfoster-r7 This was to skip the testing when the remote DB was enabled since it was not implemented at the time. Yes, the creds API was ported over to the remote DB. See PR #10176.

Reply:
Thanks for the context! :+1: